File: 백준/Bronze/1924. 2007년/2007년.py
x, y = map(int, input().split())
day = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']

if x != 1:
    for i in range(1, x):
        if i in [1, 3, 5, 7, 8, 10, 12]:
            y += 31
            
        elif i in [4, 6, 9, 11]:
            y += 30
    
        elif i in [2]:
            y += 28

print(day[y % 7])

File: 백준/Bronze/1924. 2007년/2007년.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // 2007. 1. 1 (MON)
        String[] week = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
        // 31 = {1, 3, 5, 7, 8, 10, 12} (월)
        // 30 = {4, 6, 9, 11} (월)
        // 28 = {2} (월)
        int[] days_amount = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

        int month = sc.nextInt();
        int day = sc.nextInt();
        int sum = 0;

        for (int i = 1; i < month; i++) {
            sum += days_amount[i];
        }
        System.out.println(week[(sum + day) % 7]);
    }
}